src/time/span.cr

```diff
@@ -286,6 +286,10 @@ struct Time::Span
   def self.zero
     new(0)
   end
+              
+  def zero?
+    self == typeof(self).zero
```

Comment:
Unless we create a `Zeroable` module that we just include on every type that has a `self.zero` implementation? That also allows people to juste drop in `Zeroable` on their own classes if they need to.

e.g.
```crystal
class Vector3
  def self.zero
    new 0, 0, 0
  end
  include Zeroable
end
```